Update faq for delegated consents (#166)
vippsas/vipps-login-api: vipps-login-api.md

| openid            | Scope used to request an Id-token. It provides the claim “sub” which is a unique id for the end user at that particular merchant. Note: Different merchants will get different subs for the same end user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | no                    |

> | address           | User can have up to three addresses in Vipps: home, work and other. Users' addresses are given as claims 'address' and 'other_addresses'. The claim 'address' returns the address set as 'default' for the Vipps user. And the claim 'other_addresses' returns all other addresses of the end user, if any.  We recommend that merchants fetch all addresses on a user and allow the user to choose which address to use in the relevant context. Some users will not have any registered address, in these situations the claim 'address' will be delivered, but the sub claims in address will be empty strings, e.i. "address" : {"country" : "", "street_address" : "", "address_type" : "", "formatted" : "", "postal_code" : "", "region" : "" } . If a user has information in the «Unit, floor or other details» field this will be included in the "street_address" response. The «Street address» will then be presented first before "\n" and then the contents from «Unit, floor or other details», e.g: "Suburbia 23"\nUnit B5" | yes                   |

Reviewer: If you have code-like things, you put them inside these symbols: `    `  
Then, they will look nice on the website. For example, `email_verified : true`    
Any of those variable names could go inside those ``

> | address           | User can have up to three addresses in Vipps: home, work and other. Users' addresses are given as claims 'address' and 'other_addresses'. The claim 'address' returns the address set as 'default' for the Vipps user. And the claim 'other_addresses' returns all other addresses of the end user, if any.  We recommend that merchants fetch all addresses on a user and allow the user to choose which address to use in the relevant context. Some users will not have any registered address, in these situations the claim 'address' will be delivered, but the sub claims in address will be empty strings, e.i. "address" : {"country" : "", "street_address" : "", "address_type" : "", "formatted" : "", "postal_code" : "", "region" : "" } . If a user has information in the «Unit, floor or other details» field this will be included in the "street_address" response. The «Street address» will then be presented first before "\n" and then the contents from «Unit, floor or other details», e.g: "Suburbia 23"\nUnit B5" | yes                   |

Reviewer: ```suggestion
| address           | User can have up to three addresses in Vipps: home, work and other. User addresses are given as claims `address` and `other_addresses`. The claim `address` returns the address set as default for the Vipps user. The claim `other_addresses` returns any other addresses of the end user.  We recommend that merchants fetch all addresses on a user and allow the user to choose which address to use in the relevant context. Some users will not have any registered address, in these situations the claim `address` will be delivered, but the sub claims in address will be empty strings (e.g.,  "address" : {"country" : "", "street_address" : "", "address_type" : "", "formatted" : "", "postal_code" : "", "region" : "" }) . If a user has information in the *Unit, floor or other details* field, this will be included in the *street_address* response. The *Street address* will then be presented first, before "\n". Then, the contents from *Unit, floor or other details* (e.g., "Suburbia 23"\nUnit B5") | yes                   |
```

| birthDate         | User birth date (verified with National Population Register)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | yes                   |

> | email             | User email (verified), the flag "email_verified : true" in the response can be used by merchant to confirm for each request that the email actually is verified                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | yes                   |

Reviewer: ```suggestion
| email             | User email (verified), the flag `email_verified : true` in the response can be used by merchants to confirm that the email actually is verified for each request                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | yes                   |
```

| phoneNumber       | Verified phone number (verified - the number used with Vipps)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | yes                   |

> | nin               | Norwegian national identity number (verified with National Population Register). NB: merchants need to apply for access to NIN. Go to [Who can get access to NIN and how?](vipps-login-api-faq.md#who-can-get-access-to-nin-and-how) For more information                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | yes                   |

Reviewer: ```suggestion
| nin               | Norwegian national identity number (NIN) (verified with National Population Register). NB: merchants need to apply for access to NIN. Go to [Who can get access to NIN and how?](vipps-login-api-faq.md#who-can-get-access-to-nin-and-how) For more information                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | yes                   |
```

> | delegatedConsents | Enabled consents to be collected on behalf of a merchant. This scope is only supported in Vipps login from phone number, see [Merchant delegated consents](#merchants-delegated-consents)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | yes                   |

Reviewer: ```suggestion
| delegatedConsents | Enabled consents to be collected on behalf of a merchant. This scope is only supported in Vipps login from phone number, see [Merchant delegated consents](#merchants-delegated-consents).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | yes                   |
```

* `429` status responses: Too many login requests started towards the same user at the same time. Please respect the `Retry-After` header returned.
* Most of the [general error codes](vipps-login-api.md#error-handling)

### Merchants delegated consents

#### Activation

See [How can I get started with delegatedConsents?](vipps-login-api-faq.md#how-can-I-get-started-with-delegatedConsents?).

#### Overview

For Client-Initiated Backchannel Authentication (CIBA) it is possible to initiate the authentication of an end-user through out-of-band mechanisms and collect consents on behalf of the merchant.

> 1) The Client shall make an "HTTP POST" request to the Backchannel Authentication Endpoint with scope delegatedConsents to ask for end-user authentication and on behalf of the merchant consents.

Reviewer: ```suggestion
1) The Client shall make an "HTTP POST" request to the Backchannel Authentication Endpoint with scope `delegatedConsents` to ask for end-user authentication and on behalf of the merchant consents.
```

        "confirmConsentButtonText" : "Confirm",
        "links" : {
          "termsLinkText" : "Read the terms of use.",
          "termsLinkUrl" : "https://www.vipps.no/vilkar/vilkar-privat/",
          "privacyStatementLinkText" : "Read the privacy policy.",
          "privacyStatementLinkUrl" : "https://www.vipps.no/vilkar/cookie-og-personvern/"
        },
        "timeOfConsent" : "2022-11-23T11:40:13Z",
        "consents" : [ {
          "id" : "email",
          "accepted" : true,
          "required" : true,
          "textDisplayedToUser" : "Receive offers via email"
        }, {
          "id" : "sms",
          "accepted" : true,
          "required" : false,
          "textDisplayedToUser" : "Receive offers via SMS"
        }, {
          "id" : "digital",
          "accepted" : true,
          "required" : false,
          "textDisplayedToUser" : "I would like to receive digital marketing"
        }, {
          "id" : "personal",
          "accepted" : true,
          "required" : false,
          "textDisplayedToUser" : "Get customized offers"
        } ]
      }
    }
    ```

> #### Authentication request

Reviewer: On these little sections here, we should have a space after the title and periods at the ends of the sentences. :-)
